test/distrib/bazel/test_single_bazel_version.sh

```diff
@@ -51,12 +51,31 @@ FAILED_TESTS=""
 export OVERRIDE_BAZEL_VERSION="$VERSION"
 # when running under bazel docker image, the workspace is read only.
 export OVERRIDE_BAZEL_WRAPPER_DOWNLOAD_DIR=/tmp
-bazel build -- //... "${EXCLUDED_TARGETS[@]}" || FAILED_TESTS="${FAILED_TESTS}Build "
+
+# pick the Bazel wrapper
```

Comment:
change comment to something more useful.
E.g. # bazel distribtest don't run from the repository root, so we need to explicitly use the `tools/bazel` wrapper to run the right bazel version for us